### ISSUE TITLE
Struggle with browserified version

### ISSUE THREAD

**Issue by yeti7** [spark/softap-setup-js#14]
I followed the steps to produce a browser friendly version which resulted in 1071K softap-browser.js file. When I try it with the example at the end of the readme file I'm getting following error: 

> softap-browser.js:28 Uncaught TypeError: Cannot read property 'keepAlive' of undefined

I tried to modify the sample to explicitly configure protocol as http, resulting in exception:

> softap-browser.js:188 Uncaught Error: Implemented in protocol specific class

I can't find pre-browserified version anywhere. Any suggestions how to proceed?

**Comment by brycekahle** [comment]
What command did you run to produce the browserified version?

**Comment by yeti7** [comment]
I followed the instructions from the readme verbatim:
`browserify softap.js -s SoftAPSetup -o softap-browser.js`

Since the original post I was able to find the working version produced by somebody else, but just FYI the process of browserification described in readme does not create the working client side JS.

**Comment by brycekahle** [comment]
It looks like that might be a bit outdated. Try `browserify index.js -s SoftAPSetup -o softap-browser.js`

**Comment by brycekahle** [comment]
Also, you may have been receiving that error because there were no options passed, try `new SoftAPSetup({})` instead.

**Comment by yeti7** [comment]
I generated the JS with the suggested string and passed empty options to constructor. Now it appears the script makes the initial public-key request to the domain from which the page was loaded instead of 192.168.0.1 and consequently fails with the error when not getting the expected result:

`Uncaught TypeError: Cannot read property 'b' of undefined`

I suspect the default configuration options are not being loaded.
For now I'll stick with the working version of this file, but let me know if you want me to test when this is fixed.

**Comment by betko** [comment]
I had a similar issue. I managed to fix it by running in `lib` folder: 

```
 browserify HttpSoftAP.js -s SoftAPSetup -o softap-browser.js 
```

**Comment by aml25** [comment]
What @brycekahle said about the updated change to `index.js` instead of `softap.js` is accurate. I just did this and it worked great, from the `node_modules/softap-setup/` directory.

**Comment by m-mcgowan** [comment]
The `README.md` is updated in PR #21 to give the correct instructions, and the error due to passing no options is fixed in PR #25
